nano/test_common/system.cpp

```diff
@@ -416,11 +416,10 @@ void nano::test::system::generate_receive (nano::node & node_a)
 		auto transaction (node_a.store.tx_begin_read ());
 		nano::account random_account;
 		random_pool::generate_block (random_account.bytes.data (), sizeof (random_account.bytes));
-		auto i (node_a.store.pending.begin (transaction, nano::pending_key (random_account, 0)));
-		if (i != node_a.store.pending.end ())
+		auto item = node_a.ledger.receivable_upper_bound (transaction, random_account);
+		if (item)
```

Comment:
> Returns the next receivable entry for an account greater than 'account'

Is this condition checking the same thing as before?

Reply:
It should be. Since random_accout is a random number i.e. it's not an actual account that has a receivable entry, this is finding the next-biggest account that has a receivable entry.